app/views/layouts/_footer_navbar.html.erb

```diff
@@ -9,7 +9,7 @@
     <li><%= link_to 'News', news_messages_path %></li>
     <li><%= link_to 'Administration', admin_control_path %></li>
     <% if current_user && current_user.super_admin? %>
-    <li><%= link_to 'Super Administration', users_path %></li>
+    <li><%= link_to 'User Administration', users_path %></li>
```

Comment:
That's fine I think but please double check permission is correct.  Before I tied that link only to be seen by someone with SuperAdmin role, not Admin role.